keps/prod-readiness/sig-cli/3805.yaml

```diff
@@ -0,0 +1,6 @@
+# The KEP must have an approver from the
+# "prod-readiness-approvers" group
+# of http://git.k8s.io/enhancements/OWNERS_ALIASES
+kep-number: 3805
+alpha:
+  approver: "@johnbelamaric"
```

Comment:
Should this include a `beta` approval too?

Reply:
Good catch!